projects/ngx-maplibre-gl/src/lib/map/map.component.ts

```diff
@@ -273,6 +273,9 @@ export class MapComponent implements OnChanges, OnDestroy, MapEvent {
     viewChild.required<ElementRef<HTMLDivElement>>('container');
 
   constructor() {
+
+    afterRender(() => this.mapService.clearMapElements());
```

Comment:
This was added in #189 below the afterNextRender, so I don't think it is needed, right?

Reply:
Yes `afterRender(() => this.mapService.clearMapElements());` is already being called below `afterNextRender`

Reply:
oh yes I hadn't seen it! I'll correct that